RetailSuiteAccounting/UserInterface/01a_Bookings.md

```diff
@@ -0,0 +1,424 @@
+
+- *Text*  
+Enter a posting text for each partial posting if necessary. This field can be modified.
+
+- *Currency*  
+Click the drop-down list to select the appropriate currency if necessary.
+
+- *Rate*  
+This field updates automatically when changing the currency. It can also be modified manually.
+
+
+- [POST]  
+Click this button to make each partial posting.  
+
+- [DELETE]  
+Click this button to delete a partial posting if necessary.
+
+- [CLEAR]  
+Click this button to clear all input line fields.
+
+- [SAVE]  
+Click this button to save the partial postings and record them in the selected posting period.
+
+  > [Info] When the total invoice amount has been divided, the remainder will equal 0. At this point, the [SAVE] button becomes active and can be selected.
+
+- [CANCEL]  
+Click this button to cancel the whole process.
+
+
+## Attach document
+
+*Accounting > Post > Tab POSTINGS > Button ASSIGN RECEIPT*
+
+![Attach document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument01.png "[Attach document]")
+
+- *Directory tree*  
+It displays all folders and subfolders set up in the *Documents* module. Depending on the settings, the folders and subfolders displayed may vary. By default, the following folder and subfolders are displayed:
+
+  - *New documents*  
+  This folder contains the newly uploaded documents.
+
+  - *Registered documents*  
+  This folder contains the registered documents. As soon as a new document is linked to a posting, it is moved automatically into the *Registered documents* folder and then placed in the appropriate subfolder according to the following structure:
+
+    - 10000-69999 personal account numbers in case of a debtors'/customers' document
+    - 70000-79999 personal account numbers in case of a creditors'/suppliers' document
+
+  - *Printed documents*  
+  This folder contains the printed documents.
+
+  - *New supplier receipts*  
+  This folder contains the newly uploaded supplier receipts.
+
+
+The buttons on top allow to navigate through the folders and documents.
+
+- [START PAGE]  
+Click this button to go back to the initial view of the *Attach document* window, see [Attach document](#attach-document).
+
+- [REFRESH]  
+Click this button to refresh the window, for example after making some changes.
+
+- [PARENT DIRECTORY]  
+Click this button to go directly from the current folder to its parent directory.  
+
+- [NEW DIRECTORY]  
+Click this button to create a new folder.
+
+- [UPLOAD]  
+Click this button to upload a new document. The *Upload file to* window is displayed, see [Upload file](#upload-file).
+
+- [DELETE]  
+Click this button to delete a selected document or folder.
+
+
+All information provided in columns is read-only.
+
+- [Checkbox]  
+[comment]: <> (Checkbox does not seem to be working; when selecting the checkbox in the column header, only that one and the last one are selected, not all, as it is usually. Function? Bug?)
```

Comment:
```suggestion
- [x]  
Select the checkbox to select the corresponding document or folder. If you click the checkbox in the header, all documents and folders in the list are selected.
```
[comment]: <> (Checkbox does not seem to be working; when selecting the checkbox in the column header, only that one and the last one are selected, not all, as it is usually. Function? Bug?) 
Ich würde von nem Bug ausgehen. Wir beschreiben es, wie es funktionieren sollte.